exchanges/btse/btse.go

```diff
@@ -615,3 +615,24 @@ func (b *BTSE) calculateTradingFee(ctx context.Context, feeBuilder *exchange.Fee
 func parseOrderTime(timeStr string) (time.Time, error) {
 	return time.Parse(time.DateTime, timeStr)
 }
+
+// MillionPairs returns a map of symbol names which have a IsMillion equivalent
+func (m *MarketSummary) MillionPairs() map[string]struct{} {
+	pairs := map[string]struct{}{}
```

Comment:
map[string]struct{}{} ->  map[string]bool 
 
 Instead of this: `if _, ok := mPairs[l.Symbol]; ok {` 

Can be: `if mPairs[l.Symbol] {`

Reply:
Yes, I know that 🙂 

Maybe you already understood the background here, but just in case:
There was a lot of discussion about this already in the Go community.
For existence only checks, it's lighter weight to use empty structs than bools and check existence.
( IIRC [5% faster and 10% lighter memory](https://itnext.io/set-in-go-map-bool-and-map-struct-performance-comparison-5315b4b107b) )
We're conveying the existence of a million pair, not the truthiness of it, and we don't populate keys which are false.
Leaving unfound keys to default to false as you've suggested _works_, but it's implicit instead of explicit.
Either implementation is reasonable or justifiable.

This is a hot pathway, so I chose to be lighter.

Anyway, I'll change it if you still want.

Reply:
Fixed e78a3b5d